src/Inventory/Asset/NetworkCard.php

```diff
@@ -88,6 +88,26 @@ public function prepare(): array
                 }
             }
 
+            if (property_exists($val, 'status')) {
+                switch ($val->status) {
+                    case 'up':
+                        $val_port->ifinternalstatus = '1';
```

Comment:
This way, you override `ifinternalstatus` even if it already exists; I'm not sure we want that. Same for `ifstatus` lines 103 and above.

Reply:
It's a NetworkCard which makes NetworkPorts from the `networks` sections which neither has `ifstatus` nor `ifinternalstatus` fields.

Reply:
Ho...  Indeed, yes